.isort.cfg

```diff
@@ -0,0 +1,3 @@
+[settings]
+# default venv created by intellij
+skip=venv
```

Comment:
This is currently configured in `setup.cfg`. I would rather not add another configuration file.

Reply:
oh i see.
thanks.